src/main/java/com/base/site/security/FoodService.java

```diff
@@ -0,0 +1,15 @@
+package com.base.site.security;
+
+import com.base.site.models.Food;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service("FoodService")
+public interface FoodService {
+
+    List<Food> getAllFood();
+    Food saveFood(Food food);
+    Food getFoodById(int id);
+    void deleteFoodById(int id);
```

Comment:
Use findAll(), findById(), save(), deledeById(), and delete() so we have one standard across our project.